assets/css/refills/_flex_box.scss

```diff
@@ -15,7 +15,7 @@
   box-shadow: inset 0 0 1px $flex-box-border-color, 0 2px 4px darken($base-background-color, 10%);
   display: block;
   flex: 2 2 15em;
-  margin: $flex-box-gutter;
+  margin: 0.4em 0em 0.4em 0em;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0.4em 0em` instead of `0.4em 0em 0.4em 0em`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#shorthand)